Packs/Core/Playbooks/playbook-Impossible_Traveler.yml

```diff
@@ -1447,6 +1447,72 @@ inputs:
   required: false
   description: Comment for the ticket.
   playbookInputQuery: null
+inputSections:
+- inputs:
+  - domain
+  - username
+  - ContactUserManager
+  name: Alert Management
+  description: Incident management settings and data, including escalation processes, and user engagements.
+- inputs:
+  - WhitelistedIPs
+  - AllowlistCIDR
+  - MaxMilesPerHourAllowed
+  name: Investigation
+  description: Investigation settings and data, including any deep dive incident investigation and verdict determination.
+- inputs:
+  - preInvestigationContainment
+  - AutoContainment
+  - AbuseIPDBThreshold
```

Comment:
isn't it part of the investigation?

Reply:
Yes, the 'Threshold' input is part of the investigation phase.